libs/sdk-ui-kit/src/Dialog/StylingEditorDialog/BubbleHeaderSeparator.tsx

```diff
@@ -0,0 +1,39 @@
+// (C) 2022 GoodData Corporation
+
+import { Bubble, BubbleHoverTrigger } from "../../Bubble";
+import { Typography } from "../../Typography";
+import React from "react";
+
+/**
+ * @internal
+ */
+export interface IBubbleHeaderSeparatorProps {
+    title?: string;
+    message?: string;
+}
+
+/**
+ * @internal
+ */
+export const BubbleHeaderSeparator = (props: IBubbleHeaderSeparatorProps) => {
+    const { title, message } = props;
```

Comment:
you can put this directly to the function param
({ title, message }: IBubbleHeaderSeparatorProps)